simulation/lowfi_simulation/src/main/native/include/LowFiSim/WpiSimulators/WpiMotorSim.h

```diff
@@ -0,0 +1,39 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2008-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <memory>
+
+#include "LowFiSim/MotorModel/MotorModel.h"
+#include "LowFiSim/MotorSim.h"
+#include "Simulation/PWMSim.h"
+
+namespace frc {
+namespace sim {
+namespace lowfi {
+
+class WpiMotorSim : public MotorSim {
+ public:
+  explicit WpiMotorSim(int index, MotorModel& motorModelSimulator);
+  ~WpiMotorSim();
```

Comment:
This destructor can be omitted completely since it doesn't do anything and isn't virtual.